kuma/javascript/src/ga-provider.jsx

```diff
@@ -28,6 +51,25 @@ const context = React.createContext<GAFunction>(ga);
 export default function GAProvider(props: {
     children: React.Node,
 }): React.Node {
+    /**
+     * Checks for the existence of the analytics parameter in the URL query,
+     * and sends events for every present parameter for which we have associated
+     * analytics data.
+     */
+    useEffect(() => {
+        const ids = window.location.search
+            .substr(1)
+            .split('&')
+            .map((part) => part.split('='))
+            .filter(
+                ([key, id]) => key === GA_QUERY_KEY && id in QUERY_PARAM_GA_DATA
+            );
```

Comment:
Nit: You could just do:
```javascript
const sp = new URLSearchParams(window.location.search.substr(1));
if (sp.has(GA_QUERY_KEY) && sp.get(GA_QUERY_KEY) in QUERY_PARAM_GA_DATA) {
  ga('send', QUERY_PARAM_GA_DATA[sp.get(GA_QUERY_KEY)]);
}
```
It's marginally safer since you're letting the browser APIs do the "string parsing".